service/surf/apps/users/views.py

```diff
@@ -83,3 +84,10 @@ def get(self, request, *args, **kwargs):
         if "permissions" in request.session:
             del request.session["permissions"]
         return Response({'token': token.key})
+
+
+class DeleteAccountAPIView(APIView):
+    def post(self, request, *args, **kwargs):
+        user = User.objects.get(id=request.data["id"])
+        user.delete()
```

Comment:
volgens mij doet dit een SQL `delete`. Gaat dit dan wel goed als een user bijvoorbeeld in communities zit?

Reply:
ja